pytorch_lightning/profiler/base.py

```diff
@@ -65,6 +72,18 @@ def __init__(
         self._local_rank: Optional[int] = None
         self._stage: Optional[str] = None
 
+    @abstractmethod
+    def start(self, action_name: str) -> None:
+        """Defines how to start recording an action."""
+
+    @abstractmethod
+    def stop(self, action_name: str) -> None:
+        """Defines how to record the duration once an action is complete."""
+
+    @abstractmethod
+    def summary(self) -> str:
+        """Create profiler summary in text format."""
+
```

Comment:
We don't need this. The trainer never explicitly calls `summary()` and it's redundant with `describe` already on the base class.